### DISCUSSION TITLE
Add better support for one arm exercises

### DISCUSSION THREAD

**Discussion by astashov** [astashov/liftosaur#120] [Ideas]
So that you could track left/right arm reps separately, and that properly be counted in the total/muscle group volume.

**Comment by tsvng** [comment]
In a similar vein/use case would be tracking drops sets or myoreps/rest-pause as fractions of set. 
Ex. If Ido myorep sets,I like to count 2 myorep sets as 1 weekly volume set.

Perhaps a way to implement this would be to allow us to apply a multiplier on how much each set should be worth.
Ex. a unilateral exercise could be counted as 0.5 set of weekly volume.
If I do standard set into 3 drop sets maybe I could count the standard straight set as 1 weekly set, and then 3 drop sets, each which count as 0.333 sets for a total of 2 weekly sets.

Hope my examples make sense!

**Reply by AdEmergency625** [reply]
I have been using custom exercises for single-arm variations. Maybe another way to implement this is to allow entering our set multiplier for custom exercises. I like to track my left and right exercises separately to understand my imbalances and correct that actively.

**Comment by astashov** [comment]
This is done
